Issues while using OpenCV 3.0.0-dev master branch

```
$ go run face_detect.go 
# github.com/lazywei/go-opencv/opencv
In file included from /usr/local/include/opencv2/core/core.hpp:48:0,
                 from opencv.h:16,
                 from ../opencv/cluster.go:7:
/usr/local/include/opencv2/core.hpp:47:4: error: #error core.hpp header must be compiled as C++
 #  error core.hpp header must be compiled as C++
    ^
In file included from /usr/local/include/opencv2/core.hpp:52:0,
                 from /usr/local/include/opencv2/core/core.hpp:48,
                 from opencv.h:16,
                 from ../opencv/cluster.go:7:
/usr/local/include/opencv2/core/base.hpp:48:4: error: #error base.hpp header must be compiled as C++
 #  error base.hpp header must be compiled as C++
    ^
/usr/local/include/opencv2/core/base.hpp:51:19: fatal error: climits: No such file or directory
 #include <climits>
                   ^
compilation terminated.
```

--- a/opencv/opencv.h
+++ b/opencv/opencv.h
@@ -7,16 +7,12 @@
 #ifndef _GO_OPENCV_BINDING_H_
 #define _GO_OPENCV_BINDING_H_
 
-//  OpenCV 2.4.x
 #include <opencv/cv.h>
 #include <opencv/highgui.h>
 #include <opencv2/photo/photo_c.h>
 #include <opencv2/imgproc/imgproc_c.h>
 #include <opencv2/core/core_c.h>
-#include <opencv2/core/core.hpp>
-#include <opencv2/objdetect/objdetect.hpp>
-#include <opencv2/legacy/compat.hpp>
-#include <opencv2/legacy/legacy.hpp>
+#include <opencv2/core/version.hpp>
 
 // Trackbar
 int GoOpenCV_CreateTrackbar(
@@ -38,5 +34,3 @@ unsigned GoOpenCV_FOURCC_(
 );
 
 #endif // _GO_OPENCV_BINDING_H_
-
-

